browser/ai_chat/common/constants.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_AI_CHAT_COMMON_CONSTANTS_H_
+#define BRAVE_BROWSER_AI_CHAT_COMMON_CONSTANTS_H_
+
+#include "build/buildflag.h"
+#include "components/grit/brave_components_strings.h"
+#include "ui/base/webui/web_ui_util.h"
+
+namespace ai_chat {
+constexpr char kHumanPrompt[] =
+    "\n\nHuman: "; /* Note the blank space intentionally added here */
+constexpr char kAIPrompt[] = "\n\nAssistant:";
+constexpr char kAIChatCompletionPath[] = "v1/complete";
+
+constexpr webui::LocalizedString kLocalizedStrings[] = {
```

Comment:
please put this into `.cc`. constexpr arrays of char arrays are inlined in .cc files which creates duplicates and increases binary size.

`constexpr char kString[]` is fine in header.

Reply:
Followed `/brave/components/sidebar/constants.h` example here. Fixed.

Reply:
I don't understand. Simply putting `constexpr char kString[]` complains that I need to initialize. Do you mean `extern const chat kString[]`

Reply:
Done